pkgs/development/libraries/capstone/default.nix

```diff
@@ -12,6 +12,10 @@ stdenv.mkDerivation rec {
   configurePhase = '' patchShebangs make.sh '';
   buildPhase = '' ./make.sh '';
   installPhase = '' env PREFIX=$out ./make.sh install '';
+  
+  buildInputs = [
+    pkgconfig
```

Comment:
I can't say I really fully understand the difference between `buildInputs` and `nativeBuildInputs`. Could you elaborate?